docs/submission/submitting-molecular-data.md

```diff
@@ -60,39 +60,39 @@ storage.url=https://score.qa.argo.cancercollaboratory.org
 
 ### Understanding Song payload fields
 
-Song accepts data in JSON format, which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
+Song accepts metadata in JSON format (also referred as a `Song payload`), which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
 
 The data fields can be broken down into four main sections: `experiment`, `sample`, `read groups` and `files`.
 
 > Note: \*\* denotes a required field. These must be provided as part of the metadata payload or it will immediately fail validation upon submission.
 
-#### **Experiment:** The experiment section contains details that are relevant to the experimental requirements imposed during sequencing.
+**Experiment:** The experiment section contains details that are relevant to the experimental requirements imposed during sequencing.
 
 - `**studyId`: Corresponds to your ARGO `Program ID`. This is the unique id that is assigned to your program. If you have logged into the ARGO Data Platform, this is the Program Id that you see in the Program Services area. For example, PACA-CA is a Program ID.
 - `**submitter_sequencing_experiment_id`:The unique identifier of the sequencing experiment.
 - `**platform`: The sequencing platform type that was used to generate the submitted data files.
 - `platform_model`: The exact model number of the sequencing machine used.
 - `sequencing_center`: The sequencing center the analysis was performed at.
-- `**experimental_strategy`: Descriptor of the read domain experiment method; sometimes referred to as library strategy.
+- `**experimental_strategy`: Descriptor of primary experimental method. For sequencing data it refers to how the sequencing library was made. Permissible values: WGS, WXS, RNA-Seq, Bisulfite-Seq etc.
 - `sequencing_date`: Date sequencing was performed.
 - `read_group_count`: Number of read groups submitted as part of the raw molecular file.

```

Comment:
`read_group_count` is not under `experiment` section.

Reply:
i added a "main body" section